proposals/dao/optimisticTimelock.ts

```diff
@@ -27,20 +28,58 @@ async function run(addresses, oldContracts, contracts, logging = false) {
   await core.grantRole(role, optimisticTimelock.address);
 }
 
-async function teardown(addresses, oldContracts, contracts, logging) {}
+async function teardown(addresses, oldContracts, contracts, logging) {
+
+    const {
+        tribalChiefOptimisticTimelock
+    } = contracts;
+
+    const {
+        tribalChiefOptimisticMultisigAddress
+    } = addresses;
+    
+    await hre.network.provider.request({
+        method: 'hardhat_impersonateAccount',
+        params: [tribalChiefOptimisticMultisigAddress],
+    });
+
+    await tribalChiefOptimisticTimelock.queueTransaction(
+        '0x956F47F50A910163D8BF957Cf5846D573E7f87CA',
```

Comment:
nit, but for future should add a comment explaining what this address was (I looked it up on etherscan ofc)

Reply:
ah good call, I didn't use the feiAddress var because I wanted to use the exact calldata used by OA